.github/workflows/deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: deploy
+
+on:
+  push:
+    branches-ignore:
```

Comment:
Is this `branches-ignores` necessary? From my understanding, if you just specify `tags`, the workflow would only run on tag push, and not on branch push.

Reply:
You are right.
I forgot to remove the line.